### ISSUE TITLE
SkAnimatedImage does not use custom font loaded stream in animations

### ISSUE THREAD

**Issue by Sniperxx** [skia4delphi/skia4delphi#203]
Hello,
I have a Json lottie example animation with some text (string) in it
For that text I've used a custom font called Digital Disolation.
So, Under Windows 64bit and because the font is intallled in the system the animation uses the font and
renders perfectly:
https://i.ibb.co/5ThpX3b/Under-Windows-64-Bit.png
But under android the font is not installed in the system, so i have to load it from resource and register it first 

```delphi
  RS := TResourceStream.Create(HInstance, 'Font', RT_RCDATA);
  Try
    TSkDefaultProviders.RegisterTypeface(RS);
  finally
    RS.Free;
  end;
```
After that, I'm loading the animation from a resource and playing it:
```delphi
  RS := TResourceStream.Create(HInstance, 'Lottie', RT_RCDATA);
  Try
    SkAnimatedImage1.LoadFromStream(RS);
  finally
    RS.Free;
  end;
```
In this case using the font in a TLabel works fine, but the animation did not use the font at all!! but used Arial or something similar instead!
https://i.ibb.co/D1vh5gp/Under-Android-64-Bit.jpg

[An FMX Sample Source included.](https://www.mediafire.com/file/b0r5b786tky2ja4/Lottie+and+Font+Test.rar/file)

Is there  something I'm missing?
btw, I know about saving the text as Glyphs in After Effects Bodymovin Plugin but i am facing another problem there..
Thanks in advance.

**Comment by viniciusfbb** [comment]
You are right! We are just setting the default resource provider, not the FontManager in the lottie module (called skottie in source) 
https://github.com/google/skia/blob/1d38c75ea841ca1152eba50bb9cf98915837b4c0/modules/skottie/include/Skottie.h#L126
https://github.com/skia4delphi/skia4delphi/blob/68ed3a057357a68d6c187c77a6802cfe2917d877/Source/FMX/Skia.FMX.pas#L3951

**Comment by Sniperxx** [comment]
So how to solve this for now?
what shall i do here:
`LSkottie := TSkottieAnimation.MakeFromStream(AStream, TSkDefaultProviders.Resource); `

**Comment by viniciusfbb** [comment]
It does not have fast workaround. The main change to fix it is in Skia library (C++ side), exposing the Skottie.SetFontManager, and then implementing it in delphi side...
We will fix it in next version, in some weeks.

**Comment by viniciusfbb** [comment]
Fixed in our new release [v5.0.0](https://github.com/skia4delphi/skia4delphi/releases)
